guacamole-docker/bin/start.sh

```diff
@@ -536,6 +536,21 @@ END
 
 }
 
+##
+## Adds properties to guacamole.properties which configure the TOTP two-factor
+## authentication mechanism.
+##
+associate_totp() {
+    # Update config file
+    set_property "totp-issuer"    "$TOTP_ISSUER"
+    set_property "totp-digits"    "$TOTP_DIGITS"
+    set_property "totp-period"    "$TOTP_PERIOD"
+    set_property "totp-mode"      "$TOTP_MODE"
```

Comment:
I think we want `set_option_property`, right, since all of these already have default values?

Reply:
For sure, I just then pushed the required fix.
Thank you @necouchman 👍